scripts/data/getProtectedAreasWDPA.yml

```diff
@@ -1,8 +1,15 @@
 script: getProtectedAreasWDPA.R
-name: Get protected areas (DEPRECATED - please use getWDPA.py instead)
+name: Get protected areas (DEPRECATED)
+lifecycle:
+  status: deprecated
+  message: |
+    This script is deprecated and will be removed in a future release.
+    Please use the new Python script `getWDPA.py` instead.
```

Comment:
refer to the name of the script that users see instead of the file name, or both.